README.md

```diff
@@ -127,6 +129,16 @@ is to *not* use repoload with non-UTF8 character sets.
 
 It's planned to fix this issue in the next version of repoload.
 
+### Interactive change view with fzf
+
+With fzf (A command-line fuzzy finder) it's easy to build a simple CR viewer.
+To browse all open CR's load them with `repoload changes` and use fzf's preview
+function to call `repoload show` for the selected CR.
+
+Install fzf: https://github.com/junegunn/fzf
```

Comment:
I think it's good style to have a descriptive sentence/paragraph before every code block that acts as a introduction for the following commands. So can you move this `Install fzf ...` belwo the code block and have a small text paragraph here.